mysql-replicator-streams/pom.xml

```diff
@@ -10,4 +10,19 @@
     </parent>
     <artifactId>mysql-replicator-streams</artifactId>
     <name>mysql-replicator-streams</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.dropwizard.metrics</groupId>
+            <artifactId>metrics-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>26.0-jre</version>
```

Comment:
If guava is needed, add it to the base pom and have the version there. So that this version is not duplicated over more than one module.